src/engraving/libmscore/measure.cpp

```diff
@@ -4115,12 +4077,77 @@ static bool hasAccidental(Segment* s)
 }
 
 //---------------------------------------------------------
-//   computeMinWidth
-//    sets the minimum stretched width of segment list s
-//    set the width and x position for all segments
+//      minSysTicks
+//      returns the shortest note/rest in the system
+//---------------------------------------------------------
+
+Fraction Measure::minSysTicks()
+{
+    Fraction minTicks = Fraction(10, 1);
+    for (MeasureBase* mb : system()->measures()) {
```

Comment:
if you're looping through the measures of the system here, I don't understand the reason why the methods is here.

I believe this method should be moved to the System class then